packages/loot-core/src/client/queries.ts

```diff
@@ -1,17 +1,31 @@
 // @ts-strict-ignore
 import { parse as parseDate, isValid as isDateValid } from 'date-fns';
 
+import {
+  type SheetNames,
+  type Binding,
+  type SheetFields,
+} from '../../../desktop-client/src/components/spreadsheet';
 import {
   dayFromDate,
   getDayMonthRegex,
   getDayMonthFormat,
   getShortYearRegex,
   getShortYearFormat,
 } from '../shared/months';
-import { q } from '../shared/query';
+import { q, type Query } from '../shared/query';
 import { currencyToAmount, amountToInteger } from '../shared/util';
+import { type CategoryEntity, type AccountEntity } from '../types/models';
+import { type LocalPrefs } from '../types/prefs';
+
+const parametrizedField =
```

Comment:
I think this utility function should be in the spreadsheet file

Reply:
good point!